### PR TITLE
fix: QnA import with global auth 

### PR DESCRIPTION
**Pull request by zhixzhan**
## Description

- QnA import with global auth
- Replace 'KB' with 'knowledge base'

## Task Item
#minor 

## Screenshots
Screen Shot 2021-07-08 at 3 25 54 PM
Screen Shot 2021-07-08 at 3 25 42 PM

### CODE REVIEW

**Comment by boydc2014** [comment]
Looks perfect ! can we fix the UT as well?

**Comment by zhixzhan** [comment]
> Looks perfect ! can we fix the UT as well?

@boydc2014 All test have passed.

File: Composer/packages/client/src/components/QnA/CreateQnAModal.tsx [hunk 1]
```diff
@@ -535,6 +444,9 @@ export const CreateQnAModal: React.FC<CreateQnAModalProps> = (props) => {
           </div>
         </div>
         <DialogFooter>
+          <div style={{ float: 'left' }}>
+            <PersonalCard />
```
Comment (benbrown) [inline Composer/packages/client/src/components/QnA/CreateQnAModal.tsx:448]: Should probably call this "PersonaCard" as "Persona" is the underlying component name.

**Review by benbrown — APPROVED** [review]

File: Composer/packages/client/src/components/QnA/CreateQnAModal.tsx [hunk 2]
```diff
@@ -167,54 +155,10 @@ export const CreateQnAModal: React.FC<CreateQnAModalProps> = (props) => {
...
-    }
-  }, [tenantId]);
-
-  useEffect(() => {
-    if (token) {
+    if (isAuthenticated) {
       setAvailableSubscriptions([]);
       setSubscriptionsErrorMessage(undefined);
-      getSubscriptions(token)
+      getSubscriptions(currentUser.token)
```
Comment (srinaath) [inline Composer/packages/client/src/components/QnA/CreateQnAModal.tsx:161]: Let's use await here

**Pull request merged**